packages/react/src/primitives/SelectField/SelectField.tsx

```diff
@@ -10,6 +10,31 @@ import { SelectFieldProps, Primitive } from '../types';
 import { splitPrimitiveProps } from '../shared/styleUtils';
 import { useStableId } from '../shared/utils';
 
+interface SelectFieldChildrenProps {
+  children?: React.ReactNode;
+  options?: SelectFieldProps['options'];
+}
+
+const selectFieldChildren = ({
+  children,
+  options,
+}: SelectFieldChildrenProps) => {
+  if (children) {
+    if (options?.length) {
```

Comment:
```suggestion
  if (children && Array.isArray(options)) {
```
Do you think this would be better? I always try to avoid nested `if...else` blocks as possible as I can

Reply:
If there's a way to handle all four of these cases without nested `if...else` blocks, then yes:
1. There are `children`, but no `options`
2. There are `children` and `options`
3. There are no `children` and no `options`
4. There are no `children`, but there are `options`

Reply:
Right, my bad. I missed something there and I updated the comment there :)